src/webapp/helpers/getDeliveryRequestNeedFormatted.js

```diff
@@ -0,0 +1,17 @@
+const { isEqual } = require("lodash");
```

Comment:
did you mean to remove this duplicate file?

Reply:
whoops, yes lol